packages/frameworks/src/frameworks.ts

```diff
@@ -1383,6 +1402,7 @@ export const frameworks = [
       },
       devCommand: {
         value: 'hugo server -D -w -p $PORT',
+        placeholder: 'hugo server -D',
```

Comment:
```suggestion
        placeholder: 'hugo server -D -w',
```

Should we keep `-w`?

Reply:
I'm not sure 🤔 Here in the quickstart it's not present so I didn't put it: https://gohugo.io/getting-started/quick-start/#step-5-start-the-hugo-server